src/components/statusRows.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
```

Comment:
Can we name this file with kebab-case (`status-rows.js`) to be consistent with the other components in Doctocat? https://github.com/primer/doctocat/tree/main/theme/src/components